capabilities/assert/assert_test.go

```diff
@@ -37,8 +36,6 @@ func TestRoundTripLocationCaveats(t *testing.T) {
 	err = dagjson.Encode(node, buf)
 	require.NoError(t, err)
 
-	fmt.Println(buf.String())
```

Comment:
Could use `t.Log` - unsure if this printing was helpful for debug prior to the change. May be worth keeping around?